public/docs/swagger.yaml

```diff
@@ -35,11 +35,9 @@ info:
         # How to use the API
         This documentation provides all the resources offered and how they may be accessed, however, and API key/token must be passed as `access_token` within the header of the request.
         ## API Registration
-        You MUST register on ther [Harvest API website]() before interfacing with the API for any development.
+        You MUST register on the [Harvest API website]() before interfacing with the API for any development.
         ### API Keys
         An application programming interface key (API key) is a code passed in by computer programs calling an application programming interface (API) to identify the calling program, its developer, or its user to the Web site. Ths is done to ensure we know how the information being provided is being accessed. Remember, the api key MUST be passedin the header of the requested through the variable `access_token`.
-<!-- ReDoc-Inject: <security-definitions> -->
-
 # the domain of the service
 host: localhost
```

Comment:
it's there as a placeholder. Depending on where we end up pushing this to, dev or staging, environment, it would be different. Especially if it goes live.